code/game/dna/dna_modifier.dm

```diff
@@ -359,17 +359,21 @@
 	else
 		return ..()
 
-/obj/machinery/computer/scan_consolenew/New()
+/obj/machinery/computer/scan_consolenew/Initialize()
```

Comment:
Pretty sure you need `mapload` as a param in initialise. Like all other initialises. I might be wrong.